service/device/registry/test-steps/src/main/java/org/eclipse/kapua/service/device/registry/steps/DeviceRegistrySteps.java

```diff
@@ -115,8 +115,8 @@
 import java.util.Date;
 import java.util.HashMap;
 import java.util.HashSet;
-import java.util.List;
 import java.util.Map;
+import java.util.List;
```

Comment:
can you please put this import back, there is no need to move it two lines down 😄